### PR TITLE
Fixed a flaky auth integration test by retrying the GetUser() API call

### PR DESCRIPTION
**Pull request by rsgowman**

### CODE REVIEW

File: test/integration/auth.spec.ts [hunk 1]
```diff
@@ -332,12 +332,29 @@ describe('admin.auth', () => {
...
+          .then(async () => {
+            // Attempt to retrieve the user 3 times (with a small delay between
+            // each attempt). Occassionally, this call retrieves the user data
+            // without the lastLoginTime/lastRefreshTime set; possibly because
+            // it's hitting a different server than the login request uses.
+            let userRecord = null;
+
+            for (let i = 0; i < 3; i++) {
+              userRecord = await admin.auth().getUser('lastRefreshTimeUser');
+
```
Comment (hiranya911) [inline test/integration/auth.spec.ts:344]: Remove redundant empty line. The one after the if block is ok.

File: test/integration/auth.spec.ts [hunk 2]
```diff
@@ -332,12 +332,29 @@ describe('admin.auth', () => {
...
+              if (userRecord.metadata.lastRefreshTime) {
+                break;
+              }
+
+              await new Promise((resolve) => {
+                setTimeout(resolve, 1000 * Math.pow(2, i));
+              });
+            }
+
```
Comment (hiranya911) [inline test/integration/auth.spec.ts:353]:
```
const metadata = userRecord!.metadata;
```

Comment (rsgowman) [reply test/integration/auth.spec.ts:352]: Done.

File: test/integration/auth.spec.ts (hunk 1 above)
Comment (rsgowman) [reply test/integration/auth.spec.ts]: Done.

**Pull request merged**